src/pykeen/datasets/ogb.py

```diff
@@ -104,6 +104,41 @@ class OGBBioKG(OGBLoader):
 
     name = 'ogbl-biokg'
 
+    def _load(self) -> None:
```

Comment:
this looks like an exact duplicate of the parent class's `_load()` function. Did I miss something? Otherwise, overriding the `_make_tf` function in the child class should be enough